doc/whats_new/v1.5.rst

```diff
@@ -38,3 +38,13 @@ TODO: update at the time of the release.
 
 - |Feature| A fitted :class:`compose.ColumnTransformer` now implements `__getitem__`
   which returns the fitted transformers by name. :pr:`27990` by `Thomas Fan`_.
+
+
+:mod:`sklearn.metrics`
+......................
+
+- |Efficiency| Improve efficiency of functions `brier_score_loss`, `calibration_curve`,
```

Comment:
Can you add `:func:` to all the referenced functions?

```suggestion
- |Efficiency| Improve efficiency of functions :func:`brier_score_loss`, :func:`calibration_curve`,
```

(You likely need to rewrap the lines to be under be 88 characters per line)